script.js

```diff
@@ -0,0 +1,235 @@
+let arrCounty = [
+    {
+        id: "1",
+        name: "Україна",
+        price: "500",
+        currency: "$",
+        stylesTop: "50%",
+        stylesLeft: "84%"
+    },
+    {
+        id: "2",
+        name: "Італія",
+        price: "1500",
+        currency: "$",
+        stylesTop: "70%",
+        stylesLeft: "57%"
+    },
+    {
+        id: "3",
+        name: "Франція",
+        price: "2500",
+        currency: "$",
+        stylesTop: "58%",
+        stylesLeft: "35%"
+    },
+    {
+        id: "4",
+        name: "Іспанія",
+        price: "3000",
+        currency: "$",
+        stylesTop: "75%",
+        stylesLeft: "24%"
+    },
+    {
+        id: "5",
+        name: "Англія",
+        price: "2000",
+        currency: "$",
+        stylesTop: "42%",
+        stylesLeft: "28%"
+    }
+]
+
+
+//запускаєм додаток
+appTravels();
+
+
+function appTravels() {
+    setTimeout(() => askBudget(), 1500);
+}
+
+//запитує бюджет і запускає різні сценарії
+function askBudget(text = "Це твій космічний консультат з подорожей. Вкажи скільки ти готовий витрати на подорож у розрахунку на одну людину в доларах?") {
+    let budget = prompt(`${text}`)
+    if (!budget) {
+        let text = "Сподіваюсь помандруємо іншим разом! До зустрічі!"
+        let wordsAstronaut = "До зустрічі! =)"
+        alert(text)
+        addClassByClass("circle", "active")
+        changeText(wordsAstronaut, "words-astronaut")
+    } else {
+        if (!budget.match(/^\d+$/)) {
+            textPrompt = `Мені потрібна лише цифра і я зможу тобі порадити чудову поїздку`;
+            askBudget(textPrompt)
+        } else {
+            textPrompt = `Є! Дякую!`;
+            alert(textPrompt);
+            //відмальовуємо країни
+            setTimeout(() => setСountries(arrCounty, budget), 1500);
+            addClassByClass("circle", "active")
+            let wordsAstronaut = "Привіт! Клікни на країну в яку б ти хотів поїхати, а я перевірю чи тобі вистачить грошей."
+            changeText(wordsAstronaut, "words-astronaut")
+        }
+    };
+}
+
+
+//функція додає клас по className
+function addClassByClass(classNameElem, classNameAdd) {
+    if (classNameElem) {
+        let elem = document.querySelectorAll(`.${classNameElem}`);
+        elem.forEach(el => {
+            el.classList.add(`${classNameAdd}`);

```

Comment:
думаю так чуть проще будет )
```suggestion
            el.classList.add(classNameAdd);

```